### ISSUE TITLE
Detect if an enemy was killed.

### ISSUE THREAD

**Issue by realsama** [OuYanghaoyue/gym_contra#1]
I have been working with this repo and I really appreciate the work done so far. I am encountering a situation where my agents does not learn to kill the soldiers and other enemies early enough and I think if it can learn this if I can provide it with a reward after  a successful kill. Is there a way to detect a successful kill.

**Comment by lossv** [comment]
Yes ,it can provide a reward score while enemy have been killed, but in the few days I have not enough time to change reward function until this year end.But I think you could rewrite the reward function by yourself that  though read my source code

**Comment by realsama** [comment]
I have tried reading the code but unfortunately I can't find any part indicating if my agent just made a kill. Could you please guide me in the right direction.

**Comment by lossv** [comment]
You can use this hex code to detect to enemy statues after your agent press the button B       

like use self.rom[0x0016] in my source code 

_ENEMY_TYPE_ADDRESSES = [0x0016, 0x0017, 0x0018, 0x0019, 0x001A]
